### PR TITLE
removed deprecated entity props

### PR DESCRIPTION
**Pull request by Rugvip**
🧹

### CODE REVIEW

File: plugins/api-docs/api-report.md [hunk 1]
```diff
@@ -122,17 +121,14 @@ export function defaultDefinitionWidgets(): ApiDefinitionWidget[];
 // Warning: (ae-missing-release-tag) "EntityApiDefinitionCard" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityApiDefinitionCard: (_: {
-  apiEntity?: ApiEntity | undefined;
-}) => JSX.Element;
+export const EntityApiDefinitionCard: (_: {}) => JSX.Element;
```
Comment (freben) [inline plugins/api-docs/api-report.md:124]: well this argument could have been deleted instead

**Review by jhaals — APPROVED** [review]
🧹 Nice

File: plugins/circleci/api-report.md [hunk 2]
```diff
@@ -87,9 +87,7 @@ export const circleCIRouteRef: RouteRef<undefined>;
 // Warning: (ae-missing-release-tag) "EntityCircleCIContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityCircleCIContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityCircleCIContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/circleci/api-report.md:90]: remove props

File: plugins/cloudbuild/api-report.md [hunk 3]
```diff
@@ -149,17 +149,14 @@ export { cloudbuildPlugin as plugin };
 // Warning: (ae-missing-release-tag) "EntityCloudbuildContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityCloudbuildContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityCloudbuildContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/cloudbuild/api-report.md:152]: remove props

File: plugins/github-actions/api-report.md [hunk 4]
```diff
@@ -31,9 +31,7 @@ export enum BuildStatus {
 // Warning: (ae-missing-release-tag) "EntityGithubActionsContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityGithubActionsContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityGithubActionsContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/github-actions/api-report.md:34]: remove props

File: plugins/jenkins/api-report.md [hunk 5]
```diff
@@ -18,9 +18,7 @@ import { RouteRef } from '@backstage/core-plugin-api';
 // Warning: (ae-missing-release-tag) "EntityJenkinsContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityJenkinsContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityJenkinsContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/jenkins/api-report.md:21]: remove props

File: plugins/kafka/api-report.md [hunk 6]
```diff
@@ -12,9 +12,7 @@ import { RouteRef } from '@backstage/core-plugin-api';
 // Warning: (ae-missing-release-tag) "EntityKafkaContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityKafkaContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityKafkaContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/kafka/api-report.md:15]: remove props

File: plugins/kubernetes/api-report.md [hunk 7]
```diff
@@ -25,9 +25,7 @@ export const clusterLinksFormatters: Record<string, ClusterLinksFormatter>;
 // Warning: (ae-missing-release-tag) "EntityKubernetesContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityKubernetesContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityKubernetesContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/kubernetes/api-report.md:28]: remove props

File: plugins/lighthouse/api-report.md [hunk 8]
```diff
@@ -70,9 +70,7 @@ export const EntityLastLighthouseAuditCard: ({
 // Warning: (ae-missing-release-tag) "EntityLighthouseContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityLighthouseContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityLighthouseContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/lighthouse/api-report.md:73]: remove props

File: plugins/rollbar/api-report.md [hunk 9]
```diff
@@ -21,9 +21,7 @@ export const EntityPageRollbar: (_props: Props) => JSX.Element;
 // Warning: (ae-missing-release-tag) "EntityRollbarContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityRollbarContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityRollbarContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/rollbar/api-report.md:24]: remove props

File: plugins/techdocs/api-report.md [hunk 10]
```diff
@@ -169,9 +169,7 @@ export const EntityListDocsTable: {
 // Warning: (ae-missing-release-tag) "EntityTechdocsContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const EntityTechdocsContent: (_props: {
-  entity?: Entity | undefined;
-}) => JSX.Element;
+export const EntityTechdocsContent: (_props: {}) => JSX.Element;
```
Comment (freben) [inline plugins/techdocs/api-report.md:172]: remove props

**Review by freben — APPROVED** [review]
Sorry the comments got repetitive. But I really think that especially these props that didn't even have a name to begin with, AND uglily are written out as `_props` just to get rid of lint errors ... they need to just go away :)

approve to unblock tho

**Pull request merged**